scripts/lib/install_nginx_rtmp.sh

```diff
@@ -42,6 +42,12 @@ rtmp {
             hls_playlist_length 60;
             # disable consuming the stream from nginx as rtmp
             deny play all;
+            push rtmp://localhost:${NGINX_RTMP_PORT}/view;
+        }
+        application view {
+            live on;
+            allow publish 127.0.0.1;
+            deny publish all;
```

Comment:
Do these two lines mean only allow localhost to publish to the rtmp server?

Reply:
Yes, in this case it would be the application show that would be publishing to this endpoint. These directives also allow for external publishing through stunnel only.